designer/server/src/main/resources/defaultDesignerConfig.conf

```diff
@@ -176,9 +176,9 @@ processToolbarConfig {
         buttons: [
           { type: "generate-and-test", disabled: { archived: true }, hidden: { fragment: true } }
           { type: "test-with-form", disabled: { archived: true } }
+          { type: "test-counts" }
```

Comment:
Why didn't this reorder the buttons on the cypress screenshot?

Reply:
Good question, I assume, config changes only, didn't trigger cypress tests. I merged staging with FE changes, let's see now.

Reply:
Ok, screenshots are updated now